pediatric_rootlets/generate_figure_rootlets_and_vertebral_spinal_levels.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+#
+# This script is used to generate a figure showing the variability between spinal levels and vertebrae levels.
+# The script also computes a distance from the PMJ to the middle of each spinal level and COV. Results are saved to
+# a CSV file.
+#
+# Usage:
+#     python generate_figure_rootlets_vertebrae_spinal_levels.py -i /path/to/data_processed
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+import os
+import sys
+import glob
+import argparse
+
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import matplotlib.patheffects as pe
+
+SUBJECT_TO_AXIS = {
+    'sub-101': 1,
+    'sub-102': 2,
+    'sub-103': 3,
+    'sub-104': 4,
+    'sub-105': 5,
+}
+SUBJECT_TO_XTICKS = {
+    'sub-101': 'sub-101',
+    'sub-102': 'sub-102',
+    'sub-103': 'sub-103',
+    'sub-104': 'sub-104',
+    'sub-105': 'sub-105',
+}
```

Comment:
These dicts might be probably removed and replaced by some more elegant solution that will be agnostic to num of sub

Reply:
Resolved in [630153c](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/630153c415a3349206ff89a3da3e030986448096).

Reply:
Nice!